
#!/usr/bin/env python
PACKAGE = "rmtt_tracker"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("linear_kp",    double_t,   0, "kp for linear",   2, -10, 50)
gen.add("linear_ki",    double_t,   0, "ki for linear",   0, -10, 50)
gen.add("linear_kd",    double_t,   0, "kd for linear",   0, -10, 50)
gen.add("angular_kp",    double_t,   0, "kp for angular",   5, -10, 50)
gen.add("angular_ki",    double_t,   0, "ki for angular",   0, -10, 50)
gen.add("angular_kd",    double_t,   0, "kd for angular",   0, -10, 50)

exit(gen.generate(PACKAGE, "rmtt_tracker", "tracker_pid"))